udata/frontend/helpers.py

```diff
@@ -28,7 +28,10 @@ def static_global(filename, _burst=True, **kwargs):
     if current_app.config['DEBUG']:
         burst = time()
     else:
-        burst = pkg_resources.get_distribution('udata').version
+        try:
+            burst = pkg_resources.get_distribution('udata').version
+        except pkg_resources.DistributionNotFound:
+            burst = time()  # Useful for gouvfr tests?
```

Comment:
It depends on how you set your development environment.
If you install udata with `pip install -e` or `python setup.py`, distribution will be set.
The main problem is how to use a different cache burst for theme assets.